backend/books/views.py

```diff
@@ -76,18 +78,30 @@ def post(self, request):
 
     def isbn_search_task(self, isbn, shared_dict, query_set):
         isbn_query_set = [book.isbn_13 for book in query_set]
+        remote_book_data = self.get_remote_book(isbn)
+        book_data = {}
 
         if isbn in isbn_query_set:
-            shared_dict['books'].append(self.parseDBBookModel(query_set[isbn_query_set.index(isbn)]))
+            book_data = self.parseDBBookModel(query_set[isbn_query_set.index(isbn)])
         else:
-            self.populate_shared_dict_with_isbn_data(isbn, shared_dict)
+            book_data = self.populate_shared_dict_with_isbn_data(isbn, shared_dict)
+        
+        if remote_book_data:
```

Comment:
What would happen here if remote_book exists for a given ISBN, but for some reason we can't get it from the external API (Google books)? Seems unlikely, but was just wondering